test/test.c

```diff
@@ -38,7 +38,8 @@ static int test_ctrl(nvme_ctrl_t c)
 	static __u8 buf[0x1000];
 
 	enum nvme_get_features_sel sel = NVME_GET_FEATURES_SEL_CURRENT;
-	int ret, temp, fd = nvme_ctrl_get_fd(c);
+	int ret, temp; 
+	struct dev_handle *hnd = nvme_ctrl_get_fd(c);
```

Comment:
I prefer `hdl` as short hand for handle, but I don't have a strong opinion on this.